zio-cli/shared/src/main/scala/zio/cli/FigFontParser.scala

```diff
@@ -0,0 +1,336 @@
+package zio.cli
+
+import scala.collection.mutable
```

Comment:
Is a mutable collection necessary somewhere here?

Reply:
Just in one place: `Parser#rep`, i'll try to replace with FP using `zio.Chunk` -- it just coming to Scala all these allocations to support abstractions... need more time to calibrate the perception 🦀 😉

Reply:
It's fine as it is, I think, just wanted to check.